Makefile

```diff
@@ -268,3 +268,25 @@ PROCS := $(shell sysctl -n hw.logicalcpu)
 else
 PROCS := $(shell nproc)
 endif
+
+.PHONY: typos-install
+typos-install: ## Install typos-cli for typos target using cargo
+	cargo install typos-cli
+
+GREP_EXCLUDE := grep -v -E 'dist-|cabal-testsuite|python-'
+
+.PHONY: users-guide-typos
+users-guide-typos: ## Find typos in users guide
+	find doc -type f -name '*.rst' | $(GREP_EXCLUDE) | xargs typos
```

Comment:
Can we have a `$FIND` variable, which, kinda like `$GREP_EXCLUDE`, factors out `find . -type f -name`? (And instead of `find doc` use something like `cd doc && find .`)?

Reply:
I had tried something like that but didn't think reusing it was better than repeating `find <dir> -type f -name '*.<ext>'`, given that I wanted to vary the `<dir>` and `<ext>`.

Reply:
I will try again using your `cd doc` suggestion.

Reply:
Thanks for suggesting `cd doc`. With that, I don't have to apply invert a grep pattern to exclude folders like `dist-newstyle`. I went with that for the users guide `*.rst` files.